src/modules/expression-evaluator/src/evaluateExpression.ts

```diff
@@ -1,149 +1,146 @@
-        try {
-          const inputObject = params?.objects ? params.objects : {}
-          const property = childrenResolved[0]
-          const fallback = childrenResolved?.[1]
-          return extractProperty(inputObject, property, fallback)
-        } catch {
-          throw new Error('Problem evaluating object')
-        }
+    case 'objectProperties':
+      if (Object.entries(params).length === 0)
+        return 'No parameters received for objectProperties node'
+      try {
+        const inputObject = params?.objects ? params.objects : {}
+        const property = childrenResolved[0]
+        const fallback = childrenResolved?.[1]
+        return extractProperty(inputObject, property, fallback)
+      } catch {
+        throw new Error('Problem evaluating object')
+      }
 
-      case 'stringSubstitution':
-        const origString: string = childrenResolved[0]
-        const replacements = childrenResolved.slice(1)
-        const regex = /%([\d]+)/g // To-Do: handle escaping literal values
-        const parameters = (origString.match(regex) || []).sort(
-          (a, b) => Number(a.slice(1)) - Number(b.slice(1))
-        )
-        let i = 0
-        return parameters.reduce((outputString, param) => {
-          return outputString.replace(param, replacements[i] !== undefined ? replacements[i++] : '')
-        }, origString)
+    case 'stringSubstitution':
+      const origString: string = childrenResolved[0]
+      const replacements = childrenResolved.slice(1)
+      const regex = /%([\d]+)/g // To-Do: handle escaping literal values
+      const parameters = (origString.match(regex) || []).sort(
+        (a, b) => Number(a.slice(1)) - Number(b.slice(1))
+      )
+      let i = 0
+      return parameters.reduce((outputString, param) => {
+        return outputString.replace(param, replacements[i] !== undefined ? replacements[i++] : '')
+      }, origString)
 
-      case 'API':
-        let urlWithQuery, returnedProperty
-        try {
-          const { url, fieldNames, values, returnProperty } = assignChildNodesToQuery([
-            '', // Extra unused field for API (query)
-            ...childrenResolved,
-          ])
-          returnedProperty = returnProperty
-          urlWithQuery =
-            fieldNames.length > 0
-              ? `${url}?${fieldNames
-                  .map((field: string, index: number) => field + '=' + values[index])
-                  .join('&')}`
-              : url
-        } catch {
-          throw new Error('Invalid API query')
-        }
-        let data
-        try {
-          data = await fetchAPIdata(urlWithQuery, params.APIfetch)
-        } catch {
-          throw new Error('Problem with API call')
-        }
-        try {
-          return extractAndSimplify(data, returnedProperty, "API - Can't resolve property")
-        } catch {
-          throw new Error('Problem parsing requested node from API result')
-        }
+    case 'POST':
+    case 'GET':
```

Comment:
can leave this as three options, 'POST', 'API' 'GET' (API === GET), might be safer and would avoid need to refactor configs

Reply:
Good idea. 6f981707